src/components/CreationHolidayForm.vue

```diff
@@ -88,168 +88,150 @@
-      } else error.description = "";
-    };
-    const checkForm = () => {
-      checkholidayType();
-      checkStartingDate();
-      checkEndingDate();
-      checkDescription();
-      return (
-        (error.holidayType.length <= 0) &
-        (error.startingDate.length <= 0) &
-        (error.endingDate.length <= 0) &
-        (error.description.length <= 0)
-      );
-    };
-    const getHolidays = () => {
-      return JSON.parse(localStorage.getItem("holidays")) ?? [];
-    };
-    const create = () => {
-      if (checkForm()) {
-        const holidays = getHolidays();
-        const holiday = {
-          id: holidays.length,
-          holidayType: holidayType.value,
-          startingDate: startingDate.value,
-          endingDate: endingDate.value,
-          returningDate: returningDate.value,
-          description: description.value,
-          createdAt: new Date().getTime(),
-        };
-        holidays.push(holiday);
-        localStorage.setItem("holidays", JSON.stringify(holidays));
-        this.close();
-      }
-    };
-    const numbersOfdays = computed(() => {
-      const differenceOfStartingAndEndingDate =
-        (endingDateTimestamp.value - startingDateTimestamp.value) /
-        timestampOfOneDay;
-      return differenceOfStartingAndEndingDate > 0
-        ? differenceOfStartingAndEndingDate
-        : "";
-    });
-    const startingDateTimestamp = computed(() => {
-      return new Date(startingDate.value).getTime();
-    });
-    const endingDateTimestamp = computed(() => {
-      return new Date(endingDate.value).getTime();
-    });
-    const returningDay = computed(() => {
-      return new Date(endingDate.value).getDay();
-    });
-    const returningDate = computed(() => {
-      let date;
-      switch (endingDate.value.length > 0) {
-        case false:
-          return "";
-        case returningDay.value === 6:
-          date = new Date(endingDate.value).getTime() + timestampOfOneDay * 2;
-          return new Date(date).toISOString().substring(0, 10);
-        case returningDay.value === 0:
-          date = new Date(endingDate.value).getTime() + timestampOfOneDay;
-          return new Date(date).toISOString().substring(0, 10);
-        default:
-          return endingDate.value;
-      }
-    });
-    watch(startingDate, () => {
-      checkStartingDate();
-      if (endingDate.value.length > 0) checkEndingDate();
-    });
-    watch(endingDate, () => {
-      checkStartingDate();
-      checkEndingDate();
-    });
-    watch(description, () => {
-      checkDescription();
-    });
-    return {
-      numbersOfdays,
-      startingDateTimestamp,
-      endingDateTimestamp,
-      returningDay,
-      returningDate,
-      types,
-      holidayType,
-      startingDate,
-      endingDate,
-      description,
-      error,
-      close,
-      create,
+const close = () => {
+  emit("close");
+};
+const checkholidayType = () => {
+  if (holidayType.value === "") {
```

Comment:
`""` is a falsy value so you can just do like that.
```suggestion
  if (!holidayType.value) {
```